highlight.io/components/common/Navbar/Navbar.tsx

```diff
@@ -59,6 +59,8 @@ const Navbar = ({
 	const [isOpen, setIsOpen] = useState(false)
 	const [prevY, setPrevY] = useState(0)
 
+	hideBanner = true
```

Comment:
we could just remove the banner `<Link` but this works too :)

Reply:
My thinking was we should keep it so that we don't have to recreate all the banner code and styles at a later date